openamp/overview.rst

```diff
@@ -47,27 +47,32 @@ upstream Linux kernel. As in upstream Linux, the OpenAMP Framework's remoteproc
 infrastructure uses virtio as the transport layer/abstraction.
 
 
+.. _project-aims:
+
 ************
 Project Aims
 ************
 
-To provide a solution to cover the :ref:`AMP Fundamentals<openamp-fundamentals-work-label>`, the
-OpenAMP project is divided into the following efforts:
+The `OpenAMP Project <https://www.openampproject.org>`_ is primarily focused on two efforts:
+
+- "standardize the interactions between operating environments in a heterogeneous embedded
+  system through open source solutions for Asymmetric MultiProcessing (AMP)"
+  [`1 <https://www.openampproject.org/docs/OpenAMPProject_Charter_Approved2024AugEmailVote.pdf>`_].
+
+- "providing a clean-room open source implementation"
```

Comment:
Implementation of what? How about:
`providing a clean-room open source implementation of the above standards`

The project also provides a third effort in the `openamp-system-reference` project which I would describe as:
`a set of reference integrations into several common operating environments on multiple vendor platforms`